lib/JsModule.js

```diff
@@ -164,15 +165,9 @@ export default class JsModule {
       return path.resolve(path.dirname(pathToCurrentFile), this.importPath);
     }
 
-    // This is likely an alias that points to a package, so let's try to find
-    // its main file from its package.json file.
-    const filePath = `node_modules/${this.importPath}/package.json`;
-    const main = resolveImportPathAndMain(filePath, [])[1];
-    if (main) {
-      return `node_modules/${this.importPath}/${main}`;
-    }
-
-    return this.importPath;
+    // If all of the above fails to find a path, we fall back to using
```

Comment:
I think there is. I didn't think too broadly about it though, my time was limited for the fix.